### PR TITLE
introduce volume-populator to openstack and ovirt providers

### PR DESCRIPTION
**Pull request by eslutsky**

### CODE REVIEW

File: cluster/providers/openstack/manifests/osp-volume-populator-deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+      app: openstack-populator
+  template:
+    metadata:
+      labels:
+        app: openstack-populator
+    spec:
+      serviceAccount: openstack-populator-account
+      containers:
+        - name: openstack-populator
+          image: quay.io/bzlotnik/openstack-volume-populator:latest
```
Comment (eslutsky) [inline cluster/providers/openstack/manifests/osp-volume-populator-deployment.yaml:65]: @bennyz  , we need to push this to quay.

File: cluster/providers/openstack/setup.sh [hunk 2]
```diff
@@ -4,15 +4,21 @@ set -ex
...
-kubectl apply -f cluster/providers/openstack/packstack_deployment.yml
+kubectl apply -f cluster/providers/openstack/manifests/packstack_deployment.yml
 while ! kubectl get deployment -n konveyor-forklift packstack; do sleep 10; done
 kubectl wait deployment -n konveyor-forklift packstack --for condition=Available=True --timeout=280s
 
 # deploy csi-driver-nfs
 cluster/providers/utils/deploy_csi_driver_nfs.sh "${ext_ip}"
 
+# apply openstack volume populator crds
+kubectl apply -f cluster/providers/openstack/manifests/osp-volume-populator-crd.yaml
```
Comment (liranr23) [inline cluster/providers/openstack/setup.sh:17]: should be a TODO here to remove. once the deployment will work using forklift no need to apply any roles/CRD related to OSP populator.

File: cluster/common.sh [hunk 3]
```diff
@@ -14,3 +14,6 @@ function k8s_patch_storage_profile {
     kubectl patch --type merge -p '{"spec": {"claimPropertySets": [{"accessModes": ["ReadWriteOnce"]}]}}' StorageProfile standard
 }
 
+function k8s_apply_volume_populator {
```
Comment (liranr23) [inline cluster/common.sh:17]: unnecessary

File: cluster/providers/openstack/manifests/osp-volume-populator-deployment.yaml [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+              name: http-endpoint
+              protocol: TCP
+---
+kind: VolumePopulator
+apiVersion: populator.storage.k8s.io/v1beta1
+metadata:
+  name: openstack-volume-populator
+sourceKind:
+  group: forklift.konveyor.io
+  kind: OpenstackVolumePopulator
```
Comment (liranr23) [inline cluster/providers/openstack/manifests/osp-volume-populator-deployment.yaml:82]: should be removed

File: cluster/common.sh (hunk 3 above)
Comment (eslutsky) [reply cluster/common.sh:17]: we need this for openstack

**Pull request merged**